apps/autonolas-registry/common-util/Contracts/addresses.tsx

```diff
@@ -59,71 +83,60 @@ const COMMON_TEST_ADDRESSES: Addresses = {
-    registriesManager: '0x10c5525F77F13b28f42c5626240c001c2D57CAd4',
-    serviceManagerToken: '0x1d333b46dB6e8FFd271b6C2D2B254868BD9A2dbd',
-    serviceRegistry: '0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a',
-    serviceRegistryTokenUtility: '0x6d9b08701Af43D68D991c074A27E4d90Af7f2276',
-    operatorWhitelist: '0x0338893fB1A1D9Df03F72CC53D8f786487d3D03E',
-  },
-  // optimistic
-  10: {
+  [mainnet.id]: MAINNET_ADDRESSES,
+  [optimism.id]: {
     serviceManagerToken: '0xFbBEc0C8b13B38a9aC0499694A69a10204c5E2aB',
     serviceRegistryL2: '0x3d77596beb0f130a4415df3D2D8232B3d3D31e44',
     serviceRegistryTokenUtility: '0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac',
     operatorWhitelist: '0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b',
   },
-  // gnosis
-  100: GNOSIS_ADDRESSES,
-  // polygon
-  137: POLYGON_ADDRESSES,
-  // base
-  8453: {
+  [gnosis.id]: GNOSIS_ADDRESSES,
+  [polygon.id]: POLYGON_ADDRESSES,
+  [base.id]: {
     serviceManagerToken: '0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b',
     serviceRegistryL2: '0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE',
     serviceRegistryTokenUtility: '0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5',
     operatorWhitelist: '0x3d77596beb0f130a4415df3D2D8232B3d3D31e44',
   },
-  // chiado
-  10200: {
-    serviceManagerToken: '0xc965a32185590Eb5a5fffDba29E96126b7650eDe',
-    serviceRegistryL2: '0x31D3202d8744B16A120117A053459DDFAE93c855',
-    serviceRegistryTokenUtility: '0xc2c7E40674f1C7Bb99eFe5680Efd79842502bED4',
-    operatorWhitelist: '0x6f7661F52fE1919996d0A4F68D09B344093a349d',
-  },
-  // arbitrum
-  42161: {
+  [arbitrum.id]: {
     serviceManagerToken: '0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5',
     serviceRegistryL2: '0xE3607b00E75f6405248323A9417ff6b39B244b50',
     serviceRegistryTokenUtility: '0x3d77596beb0f130a4415df3D2D8232B3d3D31e44',
     operatorWhitelist: '0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE',
   },
-  // celo
-  42220: {
+  [celo.id]: {
     serviceManagerToken: '0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5',
     serviceRegistryL2: '0xE3607b00E75f6405248323A9417ff6b39B244b50',
     serviceRegistryTokenUtility: '0x3d77596beb0f130a4415df3D2D8232B3d3D31e44',
     operatorWhitelist: '0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE',
   },
-  // polygon mumbai - testnet for polygon
-  80001: {
+  [mode.id]: MODE_ADDRESSES,
+  // Test nets
+  [polygonMumbai.id]: {
     serviceManagerToken: '0xE16adc7777B7C2a0d35033bd3504C028AB28EE8b',
     serviceRegistryL2: '0xf805DfF246CC208CD2F08ffaD242b7C32bc93623',
     serviceRegistryTokenUtility: '0x131b5551c81e9B3E89E9ACE30A5B3D45144E3e42',
     operatorWhitelist: '0x118173028162C1b7c6Bf8488bd5dA2abd7c30F9D',
   },
-  // base sepolia - testnet for base
-  84532: COMMON_TEST_ADDRESSES,
-  // arbitrum sepolia - testnet for arbitrum
-  421614: COMMON_TEST_ADDRESSES,
-  // optimistic sepolia - testnet for optimistic
-  11155420: COMMON_TEST_ADDRESSES,
-  // celo alfajores - testnet for celo
-  44787: COMMON_TEST_ADDRESSES,
+  [goerli.id]: {
```

Comment:
I believe goerli could go for good.

Reply:
Same about polygon mumbai

Reply:
goerli -> sepolia, polygonMumbai -> polygonAmoy. not urgent at all, but something to fix later.

Reply:
afaik, [goerli RPCs are now all dead](https://chainlist.org/chain/5) and [polygon mumbai](https://chainlist.org/chain/80001) too.. will add to eng meeting doc